build.psm1

```diff
@@ -407,12 +405,18 @@ Fix steps:
         return
     }
 
+    # Set $RuntimeRID as Win7-x64 or Win7-x86 so the package works on all supported Windows OS versions.
+    switch ($Runtime) {
+        'win-x64' { $RuntimeRID = 'win7-x64' }
+        'win-x86' { $RuntimeRID = 'win7-x86' }
+    }
+
     # set output options
     $OptionsArguments = @{
         CrossGen=$CrossGen
         Output=$Output
         FullCLR=$FullCLR
-        Runtime=$Runtime
+        Runtime=$RuntimeRID
```

Comment:
My feedback here was that it's better for transparency's sake that we use the same RIDs as dotnet-cli as opposed to doing black magic over their unfortunate naming. Anyone poking into RID documentation will just be confused if we try to normalize away the 7.